app/assets/stylesheets/global.css.scss

```diff
@@ -1,13 +1,13 @@
 #gradient {
-  background: red; /* For browsers that do not support gradients */
+  background:  #007eee; /* For browsers that do not support gradients */
   /* For Safari 5.1 to 6.0 */
-  background: -webkit-linear-gradient(left, #007EEE, #0095F3, #007FED);
+  background: -webkit-linear-gradient(left, #007eee, #0095f3, #007fed);
```

Comment:
Color literals like `#007eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#0095f3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#007fed` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Avoid vendor prefixes.